modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java

```diff
@@ -156,6 +156,7 @@ public void setMember(String value, String display) throws BadRequestException,
      * @throws BadRequestException
      * @throws CharonException
      */
+    @Deprecated
```

Comment:
Need to comment why this is deprecated and what are the suggested alternatives

Reply:
Added